packages/strapi-parts/src/SimpleMenu/SimpleMenu.stories.mdx

```diff
@@ -97,7 +99,18 @@ return (
   </Story>
 </Canvas>
 
-A menu can allow the user to make actions. It can redirect the user to another page or another part of the product. 
+A menu can allow the user to make actions. It can redirect the user to another page or another part of the product.
+
+### With IconButton
+
+<Canvas>
+  <Story name="with-iconbutton">
+    <SimpleMenu label="Menu" as={IconButton} icon={<FilterDropdown />}>
```

Comment:
Something is wrong with the tooltip or did I miss something? 
![Screenshot 2021-10-12 at 10 24 26](https://user-images.githubusercontent.com/7756284/136919924-14c38bc5-3f3d-4f68-a660-60fc72cbe7dc.png)

Reply:
I don't have this on my side. I think it's a race and relates to the tooltip positioning. We'll need to adjust tooltip and popover somehow at some points